Sources/TIM/TIM.swift

```diff
@@ -50,26 +51,35 @@ public final class TIM {
     ///   - configuration: TIMConfiguration
     ///   - customLogger: An optional custom logger for logging messages internally from `TIM`. Set to `nil` to disable logging.
     ///   - allowReconfigure: Controls whether you are allowed to call this method multiple times. It is **discouraged**, but possible if really needed... Default value is `false`.
-    public static func configure(configuration: TIMConfiguration, customLogger: TIMLoggerProtocol? = TIMLogger(), allowReconfigure: Bool = false) {
+    public static func configure(
+        configuration: TIMConfiguration,
+        customLogger: TIMLoggerProtocol? = TIMLogger(),
+        allowReconfigure: Bool = false,
+        customOIDExternalUserAgent: OIDExternalUserAgent? = nil
```

Comment:
Here's the main change - we now allow a custom `OIDExternalUserAgent` in the configuration of TIM, meaning it's possible to display login views in custom user agents, instead of always be dependent on the `AuthSFController` that is included in TIM right now.